p4-16/psa/PSA.mdk

```diff
@@ -2002,21 +1948,6 @@ synchronization, e.g. via PTP[^PTP] or NTP[^NTP].
 [^PTP]: <https://en.wikipedia.org/wiki/Precision_Time_Protocol>
 [^NTP]: <https://en.wikipedia.org/wiki/Network_Time_Protocol>
 
-~ TBD
- This text has been written assuming that it is more important for
-timestamps to be increasing at a constant rate, with no sudden "jumps"
-due to time synchronization events.  Is this what people want from
-timestamps?
-~
-
-~ TBD
- Some time synchronization methods avoid sudden "jumps" by
-temporarily speeding up or slowing down the rate of increase by a
-small percentage, until the desired synchronization is achieved.
-Which ones?  citation?).  Would anyone mind if PSA
-implementations were allowed to do this with their timestamp values?
-~
-
```

Comment:
I am OK just deleting these TBDs with questions about timestamps.  No one has made any comment about them, nor complained about the rest of the timestamp section.  Perhaps that is because they've read it and
like it, or perhaps such details are not a concern for them.  Either way, I don't see keeping the TBDs in there helping anything.